--- festival/lib/email-mode.scm.0001~	2017-09-04 17:54:08.000000000 +0200
+++ festival/lib/email-mode.scm	2017-12-25 23:32:51.401329977 +0100
@@ -40,7 +40,7 @@
 (define (email_init_func)
  "(email_init_func)
 Called on starting email text mode."
- (voice_rab_diphone)
+ (voice_nitech_us_slt_arctic_hts)
  (set! email_previous_t2w_func token_to_words)
  (set! english_token_to_words email_token_to_words)
  (set! token_to_words english_token_to_words)
@@ -67,13 +67,13 @@ Email spcific token to word rules."
    ((and (string-matches name ">")
          (string-matches (item.feat token "whitespace") 
 			 "[ \t\n]*\n *"))
-    (voice_cmu_us_awb_cg)
+    (voice_nitech_us_bdl_arctic_hts)
     nil ;; return nothing to say
    )
    (t  ;; for all other cases
      (if (string-matches (item.feat token "whitespace") 
 			 ".*\n[ \n]*")
-	 (voice_rab_diphone))
+	 (voice_nitech_us_slt_arctic_hts))
      (email_previous_t2w_func token name))))
 
 (set! tts_text_modes
--- festival/lib/sable-mode.scm.0001~	2017-09-04 17:54:08.000000000 +0200
+++ festival/lib/sable-mode.scm	2017-12-25 23:40:20.637195449 +0100
@@ -107,10 +107,10 @@
   (set! sable_vol_context nil)
   (set! sable_vol_type 'no_change)
   (set! sable_vol_factor 1.0)
-  (set! sable_current_language 'britishenglish)
+  (set! sable_current_language 'americanenglish)
   (set! sable_unsupported_language nil)
   (set! sable_language_stack nil)
-  (set! sable_current_speaker 'voice_kal_diphone)
+  (set! sable_current_speaker 'voice_nitech_us_bdl_arctic_hts)
   (set! sable_speaker_stack nil)
 )
 
@@ -468,7 +468,7 @@ SABLE mode token specific analysis."
   "(sable_init_func)
 Initialisation for SABLE mode"
   (sable_init_globals)
-  (voice_kal_diphone)
+  (voice_nitech_us_slt_arctic_hts)
   (set! sable_previous_elements xxml_elements)
   (set! xxml_elements sable_elements)
   (set! sable_previous_token_to_words english_token_to_words)
