--- ./src/modules/Text/text_modes.cc.gcc	2012-10-14 13:08:54.000000000 +0800
+++ ./src/modules/Text/text_modes.cc	2012-10-14 13:09:09.000000000 +0800
@@ -59,8 +59,8 @@
 void tts_file_user_mode(LISP filename, LISP params)
 {
 
-    volatile EST_String tmpname = make_tmp_filename();
-    volatile EST_String inname = (EST_String)get_c_string(filename);
+    EST_String tmpname = make_tmp_filename();
+    EST_String inname = (EST_String)get_c_string(filename);
     volatile EST_String filter;
     volatile EST_TokenStream ts;
     volatile LISP func;
